linter.py

```diff
@@ -37,7 +37,17 @@ class Sass(NodeLinter):
     regex_error = re.compile(
         r'^(\w*)Error: (?P<msg>.*)'
     )
-    tempfile_suffix = '-'
+    tempfile_suffix = {
+        'scss': 'scss',
```

Comment:
Wondering if we need all of these.

Reply:
I was trying to be defensive, scss/sass are not provided by sublimetext core. It depends on syntax plugin providers to name them.

Reply:
css part can be reduced to only support "CSS" core syntax.

Reply:
Wait a minute, I did that by guessing what’s wrong. Maybe sublimelinter already normalized syntax name to lower case. I will check again when I am on a computer.

Reply:
Thanks!

Reply:
Yes, my bad. SublimeLinter does normalise to lower case.

https://github.com/SublimeLinter/SublimeLinter/blob/2cccb40d4d9fdca6a7296329d2dd1a16bed6d6c2/lint/util.py#L59
```
                'syntax_map', {}).get(view_syntax, '').lower()
```